docs/source/realworld.rst

```diff
@@ -69,10 +69,11 @@ mean that the SQL files themselves are no longer valid SQL code, because
 they now contain these configured *placeholder* values, intended to
 improve modularity.
 
-Sqlfluff allows limited templating using both of the methods outlined
-above, to allow you to still lint those SQL files as part of your CI/CD
-pipeline (which is great 🙌), rather than waiting until you're in production
-(which is bad 🤦, and maybe too late).
+Sqlfluff supports both of the templating methods outlined above,
+as well as `dbt`_ projects, to allow you to still lint these
```

Comment:
@alanmcruickshank can you weigh in on docs style?  should `dbt` be backtick'ed, and should `sqlfluff` be in backticks consistently?

Reply:
I think if it's a name of a project, it shouldn't be backticked, and it should be capitalised for the default of that project, which means **dbt** and **SQLFluff** (although I don't think I've ever actually communicated that anywhere). If it's referencing a command line command, then it *should* be backticked and always lowercase.

@sethwoodworth Do you know where the best place to communicate style things like this would be?